packages/ui-library/src/components/select-field/stories.js

```diff
@@ -34,30 +47,21 @@ Factory.parameters = {
 	controls: { disable: false },
 };
 Factory.args = {
-	id: "select-field",
-	name: "name",
+	id: "select-field-0",
+	name: "name-0",
 	value: "1",
-	options: [
-		{ value: "1", label: "Option 1" },
-		{ value: "2", label: "Option 2" },
-		{ value: "3", label: "Option 3" },
-		{ value: "4", label: "Option 4" },
-	],
+	children: options.map( option => <SelectField.Option key={ option.value } { ...option } /> ),
```

Comment:
Isn't using `options` the easiest to showcase?

Reply:
Yes, replaced.